src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,74 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
+    private static final String NUMBER_SEPARATOR = ",";
+    private static final int DECREMENT = 1;
+    private static final CandidateValidator CANDIDATE_VALIDATOR = new CandidateValidator();
```

Comment:
let's remember about abstraction) And better move this filed to the method where we use it
```suggestion
    private Predicate<Candidate> candidateValidator = new CandidateValidator();
```